src/main/java/uk/gov/hmcts/juror/api/moj/report/AbstractReport.java

```diff
@@ -308,12 +320,23 @@ public ConcurrentHashMap<String, AbstractReportResponse.DataTypeValue> loadStand
             "court_name", AbstractReportResponse.DataTypeValue.builder()
                 .displayName("Court Name")
                 .dataType(String.class.getSimpleName())
-                .value(
-                    poolRequest.getCourtLocation().getName() + " (" + poolRequest.getCourtLocation().getLocCode() + ")")
+                .value(getCourtNameString(poolRequest.getCourtLocation()))
                 .build()
         ));
     }
 
+    protected String getCourtNameString(CourtLocationRepository courtLocationRepository, String locCode) {
+        Optional<CourtLocation> courtLocation = courtLocationRepository.findByLocCode(locCode);
+        if (courtLocation.isEmpty()) {
+            throw new MojException.NotFound("Court not found", null);
+        }
+        return getCourtNameString(courtLocation.get());
+    }
+
+    public String getCourtNameString(CourtLocation courtLocation) {
+        return courtLocation.getName() + " (" + courtLocation.getLocCode() + ")";
```

Comment:
This doesnt format as standard but can leave as a FE issue unless we use the code I had in that service class using stringutils

Reply:
as discussed, we can leave this as is for now